tests/spread/general/entrypoint-service/rockcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: entrypoint-service-test
+version: latest
+base: ubuntu@22.04
+summary: A ROCK with an entrypoint-service field
+description: A ROCK with an entrypoint-service field
+license: Apache-2.0
+entrypoint-service: success
+services:
+  success:
+    override: replace
+    command: "/usr/bin/echo Success"
+    startup: enabled
```

Comment:
Just for the sake of being consistent with Pebble's MO:
 - the service name should allude to an actual service/process/daemon (e.g. "nginx", "controller", etc.). In this case, something like `test-service` fits the purpose
 - the `command` should be more configurable so that we can actually test the image in `task.yaml`. so I would suggest something like `command: echo [ foo ]`